docs/specs/code/EXPORT_TRANSACTION_HISTORY_TO_FILE.md

```diff
@@ -74,16 +76,24 @@ Now users should be able to chose the time period to export transaction for, lik
 
 Export dialog now contains additional elements to select a date-period:
 
-![image](https://user-images.githubusercontent.com/5585355/50346357-31534800-0543-11e9-949a-873bcae9b854.png)
+![image](https://user-images.githubusercontent.com/5585355/50401307-9eaee500-079e-11e9-8972-a8162a3a5982.png)
```

Comment:
I would avoid having two components that refers to the same constrain. It could be confusing if I change the dates and then I realized that I actually want a specific period - should it should clean up the other values? -. 

I think maybe we could avoid that and use something similar to what google has - I would change the periods to what you said though-.

<img width="815" alt="screen shot 2018-12-24 at 11 13 07 am" src="https://user-images.githubusercontent.com/1622112/50403512-309dfa00-076d-11e9-9ce3-502f7c80039f.png">
<img width="460" alt="screen shot 2018-12-24 at 11 13 13 am" src="https://user-images.githubusercontent.com/1622112/50403513-309dfa00-076d-11e9-8547-8552b7f0b9b8.png">

Reply:
Yeah, my idea was something like this (it's in the spec too):

![image](https://user-images.githubusercontent.com/5585355/50404344-cd858500-07b7-11e9-9c66-bf64474ef26b.png)

So user can set custom dates only if explicitly selected "Custom". And if user selected some dates, but then selected "last month" again - dates in the fields are changed and selectors are disabled.

Will look for some libraries that do something similar.

Reply:
nice, so we are already on the same page :D